docs/configuration.md

```diff
@@ -447,6 +449,65 @@ The possible values are: `gtc` (default), `fok` or `ioc`.
     This is ongoing work. For now, it is supported only for binance and kucoin.
     Please don't change the default value unless you know what you are doing and have researched the impact of using different values for your particular exchange.
 
+### Understand trading_mode
```

Comment:
I think all "leverage/short" relevant settings should go to a seperate file (i think we've got `leverage.md`, iirc).
That makes it clear and distinct where they belong (and we can use ONE disclaimer at the very top of that page - instead of distributing them around different files).